src/showplanner/ImporterModal.tsx

```diff
@@ -11,6 +11,22 @@ interface ImporterProps {
 // TODO: This needs updating to actually either provide the weighting channel values (less preferred)
 // or update the importer to work that out itself.
 export function ImporterModal(props: ImporterProps) {
+  // Add support for closing the modal when the importer wants to reload the show plan.
+  useEffect(() => {
+    window.addEventListener(
+      "message",
+      (event) => {
+        if (!event.origin.includes("ury.org.uk")) {
+          console.log(event.origin);
+          return;
+        }
+        if (event.data === "reload_showplan") {
+          props.close();
```

Comment:
Shouldn't you, you know, actually reload the showplan?

ED: you do this in showplanner/index. Maybe worth a comment, just to clarify?

Reply:
Done